docker-compose.mongodb.yml

```
@@ -9,7 +9,7 @@ x-logging: &default-logging
 services:
   mongo:
     image: mongo:4.4.1
-    command: --keyFile=/data/mgkey --replSet=skynet
+    command: --keyFile=/data/mgkey --replSet=${SKYNET_DB_REPLICASET}
```

We can default to `skynet`:
```suggestion
    command: --keyFile=/data/mgkey --replSet=${SKYNET_DB_REPLICASET|-skynet}
```